mods/cnc/languages/rules/en.ftl

```diff
@@ -439,126 +491,170 @@ actor-bio-husk-name = Biological Lab (Destroyed)
 
 actor-miss =
    .name = Tech Center
-   .captured-desc = Provides range of vision.
-   .capturable-desc = Capture to give visual range.
+   .captured-desc = Provides range of vision
+   .capturable-desc = Capture to give visual range
+   .encyclopedia = Provides vision in a large radius when captured.
 
 actor-bio =
    .name = Biological Lab
-   .captured-desc = Provides infantry with Tiberium immunity.
-   .capturable-desc = Capture to enable Tiberium immunity for infantry.
+   .captured-desc = Provides infantry with Tiberium immunity
+   .capturable-desc = Infantry are immune to Tiberium once captured
+   .encyclopedia = Gives friendly infantry tiberium immunity when captured.
 
 actor-v19 =
    .name = Oil Derrick
-   .captured-desc = Provides additional funds.
-   .capturable-desc = Capture to receive additional funds.
+   .captured-desc = Provides additional funds
+   .capturable-desc = Capture to receive additional funds
+   .encyclopedia = The Oil Derrick produces 10 credits every 2 seconds when captured. Economically less efficient to capture than building a harvester, but never mines out.
 
 ## trees.yaml
-actor-splitblue-name = Blossom Tree (blue)
+actor-splitblue-name = Blue Blossom Tree
 actor-t03-transformable-name = (Tree that can transform into a Blossom Tree)
 actor-t13-transformable-name = (Tree that can transform into a Blossom Tree)
 
 ## vehicles.yaml
 actor-mcv =
    .name = Mobile Construction Vehicle
-   .description = Deploys into another Construction Yard.
+   .description = Deploys into a Construction Yard
       Unarmed
+   .encyclopedia = Deploying an MCV will change to its construction yard form. The MCV has more health in its deployed form, but undeploying it can be helpful to avoid infantry.
+
+    If you have a teammate, your teammate can “gift” you an MCV by killing an undeployed MCV near your base, which you can then reclaim with an engineer.
 
 actor-harv =
    .name = Harvester
    .generic-name = Harvester
-   .description = Collects Tiberium for processing.
+   .description = Collects Tiberium for processing
       Unarmed
+   .encyclopedia = Harvesters slowly mine Tiberium and deposit it in your Tiberium refinery, where it is converted into credits.
+
+    Can be sent to allied refineries to give income.
 
-notification-harvester-lost = Harvester lost.
+notification-harvester-lost = Harvester lost
 
 actor-apc =
    .name = APC
-   .description = Armed infantry transport.
-    Can attack Aircraft.
-      Strong vs Vehicles
-      Weak vs Infantry
+   .description = Armed infantry transport
+    Can attack Aircraft
+      Strong vs. Vehicles
+      Weak vs. Infantry
+   .encyclopedia = The APC can transport infantry, which can come in handy when capturing civilian structures. While the flak gun doesnt do much damage to enemy vehicles, the APC can resist enemy shots while your hum-vees deal the damage. It also functions as your primary AA unit when facing air units.
```

Comment:
`hum-vee` was changed to `humvee` elsewhere.

Search & replace for all affected usages, I won't make comments against the others.